### DISCUSSION TITLE
当设置过baseURL后，该实例需要向一个不同于baseURL的host发送请求，该如何操作

### DISCUSSION THREAD

**Discussion by manortec** [alovajs/alova#292] [General]
我遇到的场景是，我的应用除了要调用baseURL 的API之外，还要调用第三方的API，比如云存储等第三方的API服务。
这个时候，我已配置好的全局alova实例没法直接在url里填写第三方host，经测试仍然会吧baseURL加到前面。
我想，alova能不自动识别，当填入的url以http开头时，就不再把baseURL加入进来了。

目前我想到的办法就是单独的为每一个第三方host创建一个alova实例，并重新配置拦截器。请问，还有更好的方法吗。
alova真的很棒！

**Comment by JOU-amjs** [comment]
是的，每个baseURL对应一个实例是最佳实践，不过，如果拦截器相同的话，考虑复用同一个拦截器函数就好了

**Reply by manortec** [reply]
好的，感谢你的建议
